Makefile

```diff
@@ -11,17 +11,56 @@ package_name=$(app_name)
 cert_dir=$(HOME)/.nextcloud/certificates
 version+=master
 
-all: appstore
+all: dev-setup build-js-production
 
-release: appstore create-tag
+dev-setup: clean clean-dev npm-init
 
-create-tag:
-	git tag -a v$(version) -m "Tagging the $(version) release."
-	git push origin v$(version)
+dependabot: dev-setup npm-update build-js-production
+
+npm-init:
+	npm install --deps
+	cd vue/ && npm install
+
+npm-update:
+	npm update
+	cd vue/ && npm update
+
+build-js:
+	npm run dev
+	cd vue/ && npm run dev
+
+build-js-production:
+	npm install --production
```

Comment:
There are no runtime dependencies in root _package.json_, so I guess that this is not needed.

Reply:
Well soon™ we will move the stuff from vue/ to root. So lets just keep it